### DISCUSSION TITLE
How to programmatically reset a form ?

### DISCUSSION THREAD

**Discussion by salembaira** [data-driven-forms/react-forms#991] [Need help] [answered]
Hi guys,

Thank you for this awesome piece of work!!

How do you handle resetting a form after say submitting so the user can fill another one without reloading the page?

Best,

**Comment by Hyperkid123** [comment, marked as answer]
Hi @Baira.

The onSubmit function receives 3 arguments. Values, formApi and complete function. The form API does contain a `reset` function which will reset the form.

Check out this example: https://codesandbox.io/s/data-driven-forms-example-forked-i2col

**Reply by salembaira** [reply]
Thank you so much! I missed the second argument.
